framework/midi/miditypes.h

```diff
@@ -139,6 +139,11 @@ struct MidiData {
         return evts;
     }
 
+    tick_t lastChunksTick() const
+    {
+        return chunks.rbegin()->second.endTick;
+    }

```

Comment:
if clicked not on a note, here will be crash  (need check on is empty)
![image](https://user-images.githubusercontent.com/3818029/99872123-4251db80-2be8-11eb-92a7-e3c6a2cf41f7.png)

Reply:
fixed